openmetadata-ui/src/main/resources/ui/src/components/Entity/EntityRightPanel/EntityRightPanel.tsx

```diff
@@ -36,6 +42,8 @@ interface EntityRightPanelProps {
   domain?: EntityReference;
   onTagSelectionChange?: (selectedTags: EntityTags[]) => Promise<void>;
   onThreadLinkSelect?: (value: string, threadType?: ThreadType) => void;
+  viewAllPermission?: boolean;
+  customProperties?: Table['extension'];
```

Comment:
Type of `customProperties` is Extention but we are using it as entityDetails can you double check here?

Reply:
Made necessary changes